nixos/modules/services/web-apps/tt-rss.nix

```diff
@@ -604,6 +611,9 @@ let
           ''}
           ln -sf "${tt-rss-config}" "${cfg.root}/config.php"
           chmod -R 755 "${cfg.root}"
+          chmod -R 777 "${cfg.root}/${lockDir}"
+          chmod -R 777 "${cfg.root}/${cacheDir}"
+          chmod -R 777 "${cfg.root}/${feedIconsDir}"
```

Comment:
No, 777 is bad.

Reply:
you can use `ug+rwX` to only set the executable bit on directories I think.